4-WebApp-your-API/4-1-MyOrg/Client/Startup.cs

```diff
@@ -43,17 +43,8 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddOptions();
 
-            services.AddSignIn(Configuration);
-
-            // This is required to be instantiated before the OpenIdConnectOptions starts getting configured.
-            // By default, the claims mapping will map claim names in the old format to accommodate older SAML applications.
-            // 'http://schemas.microsoft.com/ws/2008/06/identity/claims/role' instead of 'roles'
-            // This flag ensures that the ClaimsIdentity claims collection will be built from the claims in the token
-            // JwtSecurityTokenHandler.DefaultMapInboundClaims = false;
```

Comment:
Hmm, wonder if we need to leave this comment?

Reply:
It depends on the version of Wilson. ?